src/ApiService/ApiService/onefuzzlib/notifications/JinjaTemplateAdapter.cs

```diff
@@ -11,4 +11,351 @@ public class JinjaTemplateAdapter {
+
+        if (JinjaTemplateAdapter.IsJinjaTemplate(template.Project)) {
+            didModify = true;
+            template = template with {
+                Project = JinjaTemplateAdapter.AdaptForScriban(template.Project)
+            };
+        } else if (attemptRender) {
+            await ValidateScribanTemplate(context!, log!, null, template.Project).IgnoreResult();
+        }
+
+        foreach (var item in template.AdoFields) {
+            if (JinjaTemplateAdapter.IsJinjaTemplate(item.Value)) {
+                template.AdoFields[item.Key] = JinjaTemplateAdapter.AdaptForScriban(item.Value);
+                didModify = true;
+            } else if (attemptRender) {
+                await ValidateScribanTemplate(context!, log!, null, item.Value).IgnoreResult();
+            }
+        }
+
+        if (JinjaTemplateAdapter.IsJinjaTemplate(template.Type)) {
+            didModify = true;
+            template = template with {
+                Type = JinjaTemplateAdapter.AdaptForScriban(template.Type)
+            };
+        } else if (attemptRender) {
+            await ValidateScribanTemplate(context!, log!, null, template.Type).IgnoreResult();
+        }
+
+        if (template.Comment != null) {
+            if (JinjaTemplateAdapter.IsJinjaTemplate(template.Comment)) {
+                didModify = true;
+                template = template with {
+                    Comment = JinjaTemplateAdapter.AdaptForScriban(template.Comment)
+                };
+            } else if (attemptRender) {
+                await ValidateScribanTemplate(context!, log!, null, template.Comment).IgnoreResult();
+            }
+        }
+
+        foreach (var item in template.OnDuplicate.AdoFields) {
+            if (JinjaTemplateAdapter.IsJinjaTemplate(item.Value)) {
+                template.OnDuplicate.AdoFields[item.Key] = JinjaTemplateAdapter.AdaptForScriban(item.Value);
+                didModify = true;
+            } else if (attemptRender) {
+                await ValidateScribanTemplate(context!, log!, null, item.Value).IgnoreResult();
+            }
+        }
+
+        if (template.OnDuplicate.Comment != null) {
+            if (JinjaTemplateAdapter.IsJinjaTemplate(template.OnDuplicate.Comment)) {
+                didModify = true;
+                template = template with {
+                    OnDuplicate = template.OnDuplicate with {
+                        Comment = JinjaTemplateAdapter.AdaptForScriban(template.OnDuplicate.Comment)
+                    }
+                };
+            } else if (attemptRender) {
+                await ValidateScribanTemplate(context!, log!, null, template.OnDuplicate.Comment).IgnoreResult();
+            }
+        }
+
+        return (didModify, template);
+    }
+
+    public async static Async.Task<(bool didModify, GithubIssuesTemplate template)> ConvertToScriban(GithubIssuesTemplate template, bool attemptRender = false, IOnefuzzContext? context = null, ILogTracer? log = null) {
```

Comment:
This was copied from `src/ApiService/ApiService/Functions/Migrations/JinjaToScriban.cs`, but I added the option render related args.